db/versions/0018.yml

```diff
@@ -0,0 +1,276 @@
+version: 18
+migrationScript: 0018-migration.sql
+downgradeScript: 0018-downgrade.sql
+methods:
+  wmworker_pool_errors_entities_load:
+    description: See taskcluster-lib-entities
+    mode: read
+    serviceName: worker_manager
+    args: partition_key text, row_key text
+    returns: table (partition_key_out text, row_key_out text, value jsonb, version integer, etag uuid)
+    body: |-
+      begin
+        return query
+        select
+          wmworker_pool_errors_entities_load.partition_key,
+          wmworker_pool_errors_entities_load.row_key,
+          entity_buf_encode(
+            jsonb_build_object(
+              'PartitionKey', encode_string_key(worker_pool_id),
+              'RowKey', encode_string_key(error_id),
+              'errorId', error_id,
```

Comment:
It looks like https://github.com/taskcluster/taskcluster/pull/3134#discussion_r447647401 was missed.